tests/core/test_expectation_suite.py

```diff
@@ -130,7 +130,6 @@ def test_expectation_suite_equivalence(baseline_suite, identical_suite, equivale
 
 def test_expectation_suite_dictionary_equivalence(baseline_suite):
     assert baseline_suite.isEquivalentTo({
-        "data_asset_name": {"datasource": "my_source", "generator": "my_generator", "generator_asset": "my_asset"},
```

Comment:
I'm surprised this still worked. Unless it's from an old test fixture, in which case I'm not surprised, and good catch.

Reply:
I think before, marshmallow was more forgiving about allowing extra fields. In the new API, you have to  be explicit if you want that to be allowed (and we don't)...so it threw an error when this was included before. Great eye. :)